recipes/itsxpress/1.7.1/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "itsxpress" %}
+{% set version = "1.7.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "2e04503cb1c4261e75672dfb7c2999ce27fac539d8918ec5ebd53392216d2164" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://files.pythonhosted.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - itsxpress=itsxpress.main:main
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True # [py27]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - pip
+    - python
+    - biopython >=1.60
+    - hmmer >=3.1
+    - bbmap
+    - vsearch
+
+test:
+  imports:
+    - itsxpress
+  commands:
+    - itsxpress --help
+
+about:
+  home: http://github.com/usda-ars-gbru/itsxpress
+  license: 'License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication'
+  license_family: 'PUBLIC-DOMAIN'
+  license_file: ''
```

Comment:
LICENSE.txt

Reply:
Would it be possible to do this on the next update?

I had omitted LICENSE.txt from the MANIFEST.in file so it was not in the Python package distribution. My conda recipe builds from Pypi and I've already submitted v1.7.2 to it. To include the LICENCE.txt file I would have to bump the software version to 1.7.3.  I've aready made the suggested changes on github https://github.com/USDA-ARS-GBRU/itsxpress/commit/f89fc5775fea709ca9e7db0d319815ffeb9014d7 so it should be in there next time.